inc/XamlApplication.cpp

```diff
@@ -0,0 +1,165 @@
+#include <winrt/Windows.System.h>
+
+#include "XamlApplication.h"
+#include <winrt/Windows.UI.Core.h>
+#include <CoreWindow.h>
+
+namespace winrt::Microsoft::Toolkit::Win32::UI::XamlHost::implementation {
+  XamlApplication::XamlApplication(winrt::Windows::Foundation::Collections::IVector<winrt::Windows::UI::Xaml::Markup::IXamlMetadataProvider> providers) {
+    for (auto provider : providers) {
+      m_providers.Append(provider);
+    }
+
+    Initialize();
+  }
+
+  XamlApplication::XamlApplication() {}
+
+  void XamlApplication::Initialize() {
+    const auto out = outer();

```

Comment:
the toolkit version has this as an `unsealed` class to enable is use as a base class. that is not needed in this context. I suggest dropping use of outer().